oak/client/application_client.h

```diff
@@ -62,6 +64,20 @@ class ApplicationClient {
     return CreateChannel(addr, channel_credentials, call_credentials);
   }
 
+  // Returns a gRPC Channel connecting to the specified address, initialised with a fixed Oak Label
+  // and the given signature.
+  static std::shared_ptr<grpc::Channel> CreateChannel(
```

Comment:
Maybe `CreatePrivateChannel` or something like that, and clarify that this does both labelling and authentication? Also, whatever name you go for, it would be nice to have consistent naming for the Rust counterpart.